### ISSUE TITLE
_settings.scss outdated

### ISSUE THREAD

**Issue by ppowalowski** [axyz/middleman-zurb-foundation#7]
Seemingly the `_settings.scss` in stylesheet folder doesn't correspond to the `_settings.scss` in stylesheets.
Is there a way of automating the copying of the most present `_settings.scss` file after bower update?

Noticed because some variables like `$h2-font-size` were missing.

**Comment by axyz** [comment]
Need to investigate if it is possible to trigger a script after bower update (but i don't think it is possible without using external scripts).

The cleanest solution in terms of maintainability should be to use an empty `_settings.scss`... by the way it could be useful to have commented settings to hack on.

being comments they do not affects the overall functionality of the application, so for now i think that it is reasonable to update the file manually (Hopefully they are not going to change it so often).

By the way I'm going to investigate possible clean solution (not involving the introduction of new actions to perform for the user when updating) which can keep `_settings.scss` in synch with the bower version.

Obviously I'm open to hear from you any suggestion about it.

Thanks for contributing

**Comment by ppowalowski** [comment]
Yeah, I also thought about it and think in the first place it should be cautious not to overwrite changes the user potentially already made in his  `_settings.scss`. 
Also I don't know about any hook scripts in bower - it doesn't seem to be appealing due to sensitivity about user environment in which it would be run. We could provide a hint in the readme to make the users aware of the issue. Thus this isn't satisfying..
